Bylaws.md

```diff
@@ -112,15 +112,15 @@ This section applies to any ballot that proposes a Final Guideline or a Final Ma
     * The Table of Contents,
     * The year in the “Copyright” information,
     * Footers with page numbers,
-    * Links to other sections within the document.    
+    * Links to other sections within the document, and
+    * Reifying relative effective dates (e.g. "6 months after publication") into absolute dates (e.g. "Month DD, YYYY").
```

Comment:
```suggestion
    * ratifying relative effective dates (e.g. "6 months after publication") into absolute dates (e.g. "Month DD, YYYY").
```

Reply:
I don't think "Ratify" is the right word here -- it means to confirm or approve, which this process isn't. I chose the word "reify" because it means to make the abstract concrete. But yeah, it's a rare word, so if someone has a better suggestion I'm open to it.

Reply:
It’s my non-native english then ;). 

Happy to endorse this as a ballot

Reply:
Same re: endorsement. 

As an alternative to "reifying": **_Converting_** relative dates (e.g. "6 months after publication") into absolute dates (e.g. "Month DD, YYYY").

Reply:
"Converting"

Reply:
Yeah that's a much clearer suggestion, thanks! Done.